### PR TITLE
Fixes to minor bugs in actions.

### PR DESCRIPTION
**Pull request by covesturtevant**
@choim-sci Please see the changes to actions workflows. Please apply to actions going forward.

- Add "**" to trigger off files changed in subfolders.
- For official tags, match the semver of the tag for the specific module. Previous grabbed the semver of the most recent tag, which could be a tag for a different module.

### CODE REVIEW

**Comment by choim-sci** [comment]
@covesturtevant, 

Agree with the changes.  A question though, build-push ymls do not care about the semver.  Do you have a reason to make the changes on them too?

**Comment by covesturtevant** [comment]
@choim-sci Thanks for taking a look. The semver changes were only made to the build-push-update workflows.

**Comment by choim-sci** [comment]
@covesturtevant, 

Oh, then, what is the change from "*" to "**"in the build-push workflows files  for?

**Comment by covesturtevant** [comment]
From the PR description above:

- Add "**" to trigger off files changed in subfolders.

File: .github/workflows/build_push_update_pub_packager.yml
Comment (choim-sci) [inline .github/workflows/build_push_update_pub_packager.yml:1]: Should be pub_packager, not pub_grouper?

**Review by choim-sci — CHANGES_REQUESTED** [review]
@covesturtevant, 
Please see the comment in build_push_update_pub_packager.yml.

**Review by choim-sci — APPROVED** [review]

**Pull request merged**